website/content/en/status/report-2023-10-2023-12/freebsd-foundation.adoc

```diff
@@ -0,0 +1,132 @@
+Some of this Foundation-sponsored work is described in separate report entries:
+
+* <<_openstack_on_freebsd,OpenStack on FreeBSD>>
+* <<_simd_enhancements_for_amd64,SIMD enhancements for amd64>>.
+
+Three new contractors started.
+Cheng Cui began working full-time on wireless networking.
+A main goal for Cheng's project is to assist Bjoern Zeeb with 802.11ac support in iwlwifi.
+Tom Jones began work to port the Vector Packet Processor (VPP) to FreeBSD.
+VPP is an open-source, high-performance user space networking stack that provides fast packet processing suitable for software-defined networking and network function virtualization applications.
+Olivier Certner joined the FreeBSD Foundation as a general FreeBSD developer.
+Some of Olivier's contributions so far include:
+
+* reviewing, fixing, and hardening several security policies aimed at limiting process visibility, policies that are based on user identity, group membership, or sub-jail membership
+* committing fixes in the login class code, including one that allowed unprivileged users to bypass resource limits
+* implementing a secure hardware fix for the Zenbleed issue affecting AMD Zen2 processors.
+
+Here is a sampling of other Foundation-sponsored work completed over the last quarter of 2023:
+
+* arm64: Add Armv8 rndr random number provider
+* net80211, LinuxKPI, and iwlwifi fixes and improvements
+* OpenSSL: updates to 3.0.11 and 3.0.12
+* Various freebsd-update fixes in preparation for 14.0
+* ssh: Update to OpenSSH 9.5p1
+* Various iommu fixes
+* Various makefs/zfs fixes
+
+Learn more about our software development work for all of 2023 at https://freebsdfoundation.org/blog/2023-in-review-software-development/.
+
+==== FreeBSD Infrastructure
+
+We approved over $100,000 for a cluster refresh that began in late 2023 and will carry over into the new year by purchasing and shipping 15 new servers to 4 racks generously donated by NYI in their new Chicago facility.
+The systems specifications were determined by the Cluster Administration team and consist of:
+
+* 5 package builders
+* 3 web servers
+* 2 package mirrors
+* 2 CI servers
+* 2 firewall/router
+* 1 admin bastion
+
+More on our 2023 infrastructure support can be found at: https://freebsdfoundation.org/blog/2023-in-review-infrastructure/.
+
+=== Continuous Integration and Workflow Improvement
+
+As part of our continued support of the FreeBSD Project, the Foundation supports a full-time staff member dedicated to improving the Project's continuous integration system and the test infrastructure.
+The full update can be found within the quarterly status report.
+
+==== Partnerships and Research
+
+In Q4 I connected with the following people, companies, and organizations: Phil Shafer, who works at Juniper Networks, and I met at All Things Open.
+He told me about the libxo library and his continuing work on related issues, like rewriting and filtering output to allow richer options that regular expressions provide.
```

Comment:
"that regular expressions provide" or "than regular expressions provide"?

Reply:
Greg says it's correct as is.